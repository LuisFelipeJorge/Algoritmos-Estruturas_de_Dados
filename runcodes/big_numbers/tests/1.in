12
sum 1 3
sum 225 225
sum 1123456789 1987654321
sum 101498473623545 10234586723
sum 1123456 1123459
big -1 1
sml 012143 1
big 1123456 -1123456
eql 123456789 123456789
sum 05 05
sum 25 113567
eql 0987 09870