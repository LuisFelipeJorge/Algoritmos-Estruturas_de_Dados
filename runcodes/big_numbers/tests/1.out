Resultado 1: 4
Resultado 2: 450
Resultado 3: 3111111110
Resultado 4: 101508708210268
Resultado 5: 2246915
Resultado 6: 0
Resultado 7: 0
Resultado 8: 1
Resultado 9: 1
Resultado 10: 10
Resultado 11: 113592
Resultado 12: 0